spec/models/post_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  context 'Post Model' do
+    before(:example) do
+      @user = User.first
+    end
+
+    subject do
+      Post.create(author: @user, title: 'Post', text: 'Lorem Ipsem')
+    end
+    before { subject.save }
+
+    it 'comments_counter and likes_counter are set by default' do
+      expect(subject).to be_valid
+    end
+
+    it 'comments_counter is >= 0' do
+      subject.comments_counter = -1
+      expect(subject).to_not be_valid
+    end
+
+    it 'likes_counter is >= 0' do
+      subject.likes_counter = -1
+      expect(subject).to_not be_valid
+    end
+
+    it 'Title can not be nil' do
+      subject.title = nil
+      expect(subject).to_not be_valid
+    end
+    it 'Last_comments return 5 last comments' do
+      (1..6).each do
+        Comment.create(text: 'post', author: @user, post: subject)
+      end
+      lasts = subject.last_comments
+      expect(lasts.length).to eq(5)
+    end
+  end
+end
```

Comment:
- [ ] You should also add tests for `increase_posts_counter` and `increase_posts_counter` [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/blog_app_validations_model_spec_n%2B1.md#:~:text=your%20models%27%20methods)
- [ ] Please also test for this validation:
`Title must not exceed 250 characters.`